docs/advanced-topics/metrics.rst

```diff
@@ -0,0 +1,154 @@
+Workflow Metrics
+================
+
+.. note:: Dockstore Workflow Metrics is currently in preview mode. Also note that the screenshots below were taken on our staging site.
+
+What are workflow metrics?
+------------------------------------
+Workflow metrics are metrics of workflow executions on a platform. Metrics include things like how often workflows succeeded or failed and the resources used during the execution. 
```

Comment:
> like how often workflows succeeded or failed and the resources used during the execution

Might be just me, but I don't think this sounds quite right -- to me, the `how often` sounds like it's applying to `resources used`.

Maybe a comma after `failed`? Or reorder to `like the resources used during the execution and how often the workflows succeeded or failed`.

Reply:
Reordered it